src/wazuh_modules/vulnerability_scanner/CMakeLists.txt

```diff
@@ -104,7 +104,10 @@ add_library(vulnerability_scanner SHARED
     )
 target_compile_options(vulnerability_scanner PRIVATE -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Woverloaded-virtual -Wunused -Wcast-align -Wformat=2)
 
-add_dependencies(database_feed compile_schemas_vd compile_schemas_input) #Add a dependency between top-level targets.
+if (NOT TARGET compile_schemas_input)
+    include(${SRC_FOLDER}/shared_modules/utils/flatbuffers/schemas/CMakeLists.txt)
```

Comment:
+1 This is another way to solve the problem. The dependency between the CMakeLists file exists, and here you are simply defining the missing target.